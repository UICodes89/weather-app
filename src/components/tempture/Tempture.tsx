import React from 'react'

export default function Tempture(props: any) {
  console.log(props);
  return (
    <div className="search-result">
      <div className="city-tempture">{props.city} <span>{props.tempture}</span></div>
      <div className="time-diff">{props.time_diff}</div>
    </div>
  )
}