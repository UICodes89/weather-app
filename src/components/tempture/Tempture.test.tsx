import React from 'react';
import { render, screen } from '@testing-library/react';
import Tempture from './Tempture';
import renderer from 'react-test-renderer';

test('renders learn react link', () => {
  render(<Tempture />);
});


//snapshot testings
test('APP Component snapshot', () => {
  const tree = renderer.create(<Tempture></Tempture>).toJSON();
  expect(tree).toMatchSnapshot()
})