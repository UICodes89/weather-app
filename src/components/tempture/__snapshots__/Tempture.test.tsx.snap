// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP Component snapshot 1`] = `
<div
  className="search-result"
>
  <div
    className="city-tempture"
  >
     
    <span />
  </div>
  <div
    className="time-diff"
  />
</div>
`;
